rest-service/manager_rest/utils.py

```diff
@@ -334,3 +336,14 @@ def validate_deployment_and_site_visibility(deployment, site):
             .format(deployment.id, deployment.visibility, site.name,
                     site.visibility)
         )
+
+
+def extract_host_agent_plugins_from_plan(plan):
+    host_agent_plugins_to_install = plan.get(
+        HOST_AGENT_PLUGINS_TO_INSTALL, [])
+
+    if not host_agent_plugins_to_install:
+        for node in plan.get('nodes', []):
+            for plugin in node.get('plugins_to_install', []):
```

Comment:
you sure this is going to return just host plugins?

Reply:
Yes, that's what I used in the dsl parser to generate host_agent_ plugins_to_install